domains/transfer/Transfer.API/API/Cvr/Api/Clients/Cvr/CvrClient.cs

```diff
@@ -15,21 +17,28 @@ public CvrClient(HttpClient client)
         this.client = client;
     }
 
-    public async Task<Root?> CvrNumberSearch(CvrNumber cvr)
+    public Task<Root?> CvrNumberSearch(CvrNumber cvr)
```

Comment:
I think this should be deleted and all the controllers that uses CvrNumberSearch should just use a list instead

Reply:
fixed here:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/0ea9743cb222e5b66335ac4ed2e7677c66104832

Reply:
Done:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/c171ad4ebc42b8efec6ee8a166d4362bc58e2068